modules/internal/LocaleModelHelpSetup.chpl

```diff
@@ -89,6 +90,20 @@ module LocaleModelHelpSetup {
     here.runningTaskCntSet(0);  // locale init parallelism mis-sets this
   }
 
+  proc helpSetupRootLocaleAPU(dst:RootLocale) {
+    var root_accum:chpl_root_locale_accum;
+
+    forall locIdx in dst.chpl_initOnLocales() with (ref root_accum) {
+      chpl_task_setSubloc(c_sublocid_any);
+      const node = new LocaleModel(dst);
+      dst.myLocales[locIdx] = node;
+      root_accum.accum(node);
+    }
+
+    root_accum.setRootLocaleValues(dst);
+    here.runningTaskCntSet(0);  // locale init parallelism mis-sets this
```

Comment:
This is identical to helpSetupRootLocaleNUMA. Is there some philosophical reason for this copy/paste, or can we update callers to just call helpSetupRootLocaleNUMA ?